rosdep/python.yaml

```diff
@@ -892,6 +894,11 @@ python-libpgm-pip:
   ubuntu:
     pip:
       packages: [libpgm]
+python2-lttngust:
```

Comment:
It's not usually useful to have different keys for different platforms. The goal of having these keys is that they give the same thing on different platforms. I assume you did this because it's not available in python3 on arch? Maybe a pip rule would be better?

Can you explain a little more about your use case?